lib/src/internet.dart

```diff
@@ -152,25 +149,4 @@ class Internet {
   ///   faker.internet.password();
   /// ```
   String password({int length = 10}) => random.string(length, min: length);
-
-  /// Generates an User Agent from Predefined Dictionary
-  /// with the given [osName] if provided.
-  /// if not provided [osName] is an empty String or ['']
-  ///
-  /// Example:
-  /// ```dart
-  ///   faker.internet.userAgent();
-  ///   faker.internet.userAgent(osName:'ios');
-  /// ```
-  String userAgent({String osName = ''}) => random
-      .element(UserAgents.fromJson(userAgentDatas)
-          .userAgents
-          .map((e) => e)
-          .where(
-            (element) =>
-                element.osName.toLowerCase().contains(osName.toLowerCase()),
-          )
-          .toList())
-      .userAgent
-      .toString();
```

Comment:
Why are these being deleted?

Reply:
Yep. My bad. It seems that I used and older fork of faker, without the userAgent and image commit. Can you omit those changes, or should I make another pull request? Only added a method in random_generator.dart

Reply:
Please make a new commit that brings those deletions back if possible.